amgut/lib/data_access/ag_data_access.py

```diff
@@ -428,14 +441,16 @@ def getConsent(self, survey_id):
             if result:
                 return {k: v for k, v in zip(colnames, result)}
 
-    def insertAGMultiple(self, ag_login_id, participant_name, field_name,
-                         field_value):
-        sql = ("insert into ag_survey_multiples (ag_login_id, "
-               "participant_name,item_name, item_value) values ('{0}','{1}',"
-               " '{2}', '{3}')").format(ag_login_id, participant_name,
-                                        field_name, field_value)
-        self.get_cursor().execute(sql)
-        self.connection.commit()
+    # NOTE: This method is vulnerable to SQL injection as-written but is not
```

Comment:
Much of the data access code is carryover from the old mod_python + Oracle deployment, and there are blocks of code that slipped through that should be removed. If this is not in use then please delete it.